### DISCUSSION TITLE
Lighthouse Redirects to Login Page Even After Successful Login (Tested Manually)

### DISCUSSION THREAD

**Discussion by nadeeshaOHRM** [GoogleChrome/lighthouse#16176] [Q&A]
I’m running into an issue where Lighthouse keeps redirecting to the login page and runs the performance test there, even though I’ve already logged in and navigated to the desired page.

Here’s what I’ve tried:

1.I log in and navigate to the target URL using Puppeteer.
2.After reaching the desired page, I start Lighthouse, but it always redirects back to the login page and performs the audit there instead.
3.I manually tested this using Chrome DevTools, logged in, navigated to the page, and then ran Lighthouse. The same issue occurs: Lighthouse redirects to the login page instead of auditing the page I’m already on.

**Comment by adamraine** [comment]
I assume you are following https://github.com/GoogleChrome/lighthouse/blob/main/docs/authenticated-pages.md

Can you share more details about your use case like what page you are using and how to reproduce the problem?

Also have you tried unchecking "Clear storage" when testing using Lighthouse in DevTools?
